### PR TITLE
Update Ranking Algorithm

### PR DESCRIPTION
**Pull request by estyxx**

### CODE REVIEW

File: backend/voting/models/ranking.py [hunk 1]
```diff
@@ -35,9 +36,15 @@ def __str__(self):
...
+class Rank(TypedDict):
+    submission_id: int
+    submission__tag: SubmissionTag
+    score: float
+
+
 class RankRequest(models.Model):
 
-    conference = models.OneToOneField(
+    conference = models.ForeignKey(
```
Comment (estyxx) [inline backend/voting/models/ranking.py:47]: I relaxed this constraint to be able to see both the old ranking and the new one with the tags in order to check them, but I think I will revert this, we don't need to have multiple rankings for one conference.

**Review by marcoacierno — APPROVED** [review]

**Pull request merged**